kythe/release/cloudbuild/bazelisk-builder-client/Dockerfile

```diff
@@ -0,0 +1,44 @@
+# Copyright 2021 The Kythe Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# A minimal image for building Kythe with Bazelisk
+
+# docker build -t gcr.io/kythe-repo/bazelisk-builder-client .
+FROM ubuntu:focal
+
+RUN apt-get update \
+        && apt-get upgrade -y \
+        && apt-get install -y --no-install-recommends \
+                curl ca-certificates \
+                # Bazel's fallback @local_jdk and needed by rules_jvm_external
+                openjdk-11-jdk-headless \
+                # Used to build the website; rules_ruby doesn't support alternate toolchains well.
+                ruby \
```

Comment:
Why do we need all this stuff in this Dockerfile just to get a bazelisk binary?

Reply:
We need curl and ca-certificates to get bazelisk.  We need the others to build at all, due to various Bazel dependencies (noted in the comments).